examples/README.txt

```diff
@@ -7,3 +7,5 @@ Gallery
 
 This gallery contains examples of the many things you can do with
 Matplotlib. Click on any image to see the full image and source code.
+
+See the sidebar to the right for a list of topics covered.
```

Comment:
I just added it in there because it took me like 2-3 weeks to realize that there was a sidebar w/ TOC haha. I can remove it if you think it's clear enough